agents-api/agents_api/env.py

```diff
@@ -88,8 +88,11 @@
 
 api_key_header_name: str = env.str("AGENTS_API_KEY_HEADER_NAME", default="X-Auth-Key")
 
+# Free tier limits
 max_free_sessions: int = env.int("MAX_FREE_SESSIONS", default=50)
 max_free_executions: int = env.int("MAX_FREE_EXECUTIONS", default=50)
+max_free_entries: int = env.int("MAX_FREE_ENTRIES", default=500)
```

Comment:
@whiterabbit1983 We should add these env vars to `docker-compose` and `env.example`